network/dns-cloud/how-to/_renew-internal-domain.mdx

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Should we delete this how-to? We already have another one that explains how to renew domain names with autorenew and there's no other way to renew it.

Reply:
autorenew is not the only way to renew. there is the manual renew too, triggered by the user.

Reply:
page deactivated